icu4c/source/i18n/measunit.cpp

```diff
@@ -2020,6 +2020,20 @@ MeasureUnit::MeasureUnit(const MeasureUnit &other)
     *this = other;
 }
 
+// FIXME(review):
+// Can someone explain this please? I don't have a good grip on C++ "universal
```

Comment:
This is the move constructor.  Rather than copying the data from `other`, the data that `other` owns is moved into the new object, with no heap allocations, etc., since the old object is expected to be destroyed.  This is what `&&` means.

Reply:
Aha: I see VSCode is also simply failing to resolve this correctly, showing the wrong documentation ("copy constructor"). Looking at the header file myself, I can see it's clearly documented ("Move constructor"), thanks for clarifying!